_sass/blocks/_revenues_forms.scss

```diff
@@ -4,34 +4,34 @@
 //
 // Styleguide blocks.revenues_forms
 
-.revenues-section-forms {
-  background-color: $white;
-  padding-top: $base-padding-large;
-  padding-bottom: $base-padding-large;
+.revenues_page-forms {
+  padding-top: $base-padding;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered padding-bottom, padding-top